packages/react-native-editor/package.json

```diff
@@ -47,7 +46,6 @@
     "node-libs-react-native": "^1.0.2",
     "patch-package": "^6.2.0",
     "postinstall-prepare": "^1.0.1",
-    "react-native": "0.61.5",
```

Comment:
We should still keep a react-native dependency here. The iOS autolinking script depends on react-native being a dependency of react-native-editor.